packages/adapters/ethers/src/client.ts

```diff
@@ -1,114 +1,34 @@
-import type { AppKitOptions, AppKitOptionsWithCaipNetworks } from '@reown/appkit'
-import {
-  SafeLocalStorage,
-  SafeLocalStorageKeys,
-  type AdapterType,
-  type CaipAddress,
-  type CaipNetwork,
-  type CaipNetworkId,
-  type ChainNamespace
-} from '@reown/appkit-common'
-import {
-  AccountController,
-  ChainController,
-  CoreHelperUtil,
-  AlertController,
-  type CombinedProvider,
-  type Connector
-} from '@reown/appkit-core'
-import {
-  EthersHelpersUtil,
-  type Provider,
-  type ProviderType,
-  type Address
-} from '@reown/appkit-utils/ethers'
-import type { AppKit } from '@reown/appkit'
-import {
-  W3mFrameHelpers,
-  W3mFrameProvider,
-  W3mFrameRpcConstants,
-  type W3mFrameTypes
-} from '@reown/appkit-wallet'
-import { ConstantsUtil as CoreConstantsUtil } from '@reown/appkit-core'
+import { AdapterBlueprint } from '@reown/appkit/adapters'
+import type { CaipNetwork } from '@reown/appkit-common'
 import { ConstantsUtil as CommonConstantsUtil } from '@reown/appkit-common'
-import {
-  CaipNetworksUtil,
-  ConstantsUtil,
-  ErrorUtil,
-  HelpersUtil,
-  PresetsUtil
-} from '@reown/appkit-utils'
+import { type CombinedProvider, type ConnectorType, type Provider } from '@reown/appkit-core'
+import { ConstantsUtil, PresetsUtil } from '@reown/appkit-utils'
+import { EthersHelpersUtil, type ProviderType } from '@reown/appkit-utils/ethers'
+import { WcConstantsUtil, WcHelpersUtil, type AppKitOptions } from '@reown/appkit'
 import UniversalProvider from '@walletconnect/universal-provider'
-import type { ConnectionControllerClient, NetworkControllerClient } from '@reown/appkit-core'
-import { WcConstantsUtil } from '@reown/appkit'
-import { EthersMethods } from './utils/EthersMethods.js'
 import { formatEther, InfuraProvider, JsonRpcProvider } from 'ethers'
-import type { PublicStateControllerState } from '@reown/appkit-core'
-import { ProviderUtil, type ProviderIdType } from '@reown/appkit/store'
 import { CoinbaseWalletSDK, type ProviderInterface } from '@coinbase/wallet-sdk'
-import { W3mFrameProviderSingleton } from '@reown/appkit/auth-provider'
-
-// -- Types ---------------------------------------------------------------------
-export interface AdapterOptions {
-  ethersConfig: ProviderType
-  defaultCaipNetwork?: CaipNetwork
-}
-
-type CoinbaseProviderError = {
-  code: number
-  message: string
-  data: string | undefined
-}
-
-interface ExternalProvider extends Provider {
-  accounts: string[]
-}
-
-declare global {
-  interface Window {
-    ethereum?: Record<string, unknown>
-  }
-}
+import type { W3mFrameProvider } from '@reown/appkit-wallet'
+import { EthersMethods } from './utils/EthersMethods.js'
 
 interface Info {
```

Comment:
We could remove this and use `Connector['info']` instead? Same for `ethers5` client too.